governance/charter.md

```diff
@@ -39,7 +39,7 @@ improve overall security in cloud native systems.
 
 ## Focus
 
-In addition to the [CNCF security-related projects](cncf-projects.md), there
```

Comment:
We could replace this with a link to the CNCF landscape filtered to TAG Security (https://landscape.cncf.io/?group=projects-and-products&view-mode=grid&tag=security)